tests/unit/common/browserInfoHelper.test.js

```diff
@@ -259,4 +260,10 @@ describe("BrowserInfoHelper", () => {
     expect(getScreenHeight()).toEqual(1021);
     expect(getScreenColourDepth()).toEqual(17);
   });
+
+  it("getDeviceId", () => {
+    const getUUID = getDeviceId();
+    expect(getUUID.split("-").length).toEqual(5);
```

Comment:
Please see https://github.com/intuit/user-data-for-fraud-prevention/pull/52#discussion_r498529292 for my thoughts on this testing strategy.

Reply:
sure